scripts/gha-encrypted/FIREGSignInInfo.h.gpg

```diff
@@ -0,0 +1,3 @@
+�
	��\��������]G4U��a�$�v7+@l$�|��v��zȎ��#�S��dq�����e���l�1��Ņ�vg��W��q�Y�P�B$Ԗs������U������3W�b�鬮�%7N����~N�s�	4�����]��*�O�,�
��8����a�F#,�jkz5+)*
```

Comment:
It would be nice to have a reference doc explaining how to update the files available somewhere in the repo. Maybe a go-link to it in the repo?

Reply:
Added a README with a pointer to the GHA instructions.

Reply:
Thank you!